### DISCUSSION TITLE
Configurable delay of the which-key popup window

### DISCUSSION THREAD

**Discussion by niklassc-xyz** [qutebrowser/qutebrowser#8138] [Ideas] [answered]
In normal mode when a partial key-combination is typed (e.g. `g`), after a delay a cheatsheet-popup opens with the available commands with that prefix (like the which-key plugins for vim).

I was not able to find a way to configure this in the [settings](https://www.qutebrowser.org/doc/help/settings.html). It would be nice to be able to change the delay, especially to set it to 0 so it pops up right away.

**Comment by Earnestly** [comment]
Do you think it's reasonable to have that display flicker every time you type a sequence?

**Reply by niklassc-xyz** [reply]
I mean I would not set it to 0 as default because it might bother most people, but it's what I found most useful in vim's which-key.
And if it's configurable, instead of setting it to 0, you could also just set it shorter or longer depending on what is best for you.

**Comment by niklassc-xyz** [comment, marked as answer]
I searched the docs again and apparently qutebrowser already has this option (`keyhint.delay`).
